ingest-app/src/cmr/ingest/data/bulk_update.clj

```diff
@@ -9,6 +9,14 @@
    [cmr.oracle.connection]
    [cmr.oracle.sql-utils :as su]))
 
+(defn generate-task-status-message
```

Comment:
good candidate for a unit test, and possibly move it to another namespace if its not going to be only message/utility functions.  Maybe the latter can be left for future updates.